client/my-sites/domains/domain-management/dns/style.scss

```diff
@@ -32,9 +32,6 @@ body.dns__body-white {
 		color: var(--studio-gray-50);
 		font-size: $font-body-small;
 		margin: 4px 16px;
-		@include break-mobile {
-			max-width: 677px;
-		}
```

Comment:
Is this change necessary?

Reply:
Yep this was probably related to the old design, with this line the initial paragraph doesn't get full width:

<img width="1097" alt="Screenshot 2025-01-09 alle 16 14 49" src="https://github.com/user-attachments/assets/f1f5ef94-2738-4343-83bb-2dbeea299f17" />

Reply:
Ah you're right, I hadn't noticed that!